tests/conftest.py

```diff
@@ -154,3 +155,8 @@ def app_spec() -> ApplicationSpecification:
     path = Path(__file__).parent / "app_client_test.json"
     path.write_text(app_spec.to_json())
     return read_spec("app_client_test.json", deletable=True, updatable=True, template_values={"VERSION": 1})
+
+
+@pytest.fixture(autouse=True, scope="session")
```

Comment:
Please add a little comment here explaining that if a unit test without debug mode i needed, this auto fixture can be manually set to false